tensorflow/core/api_def/base_api/api_def_ApplyAdam.pbtxt

```diff
@@ -82,9 +82,9 @@ END
   }
   summary: "Update \'*var\' according to the Adam algorithm."
   description: <<END
-$$lr_t := \text{learning\_rate} * \sqrt{1 - beta_2^t} / (1 - beta_1^t)$$
-$$m_t := beta_1 * m_{t-1} + (1 - beta_1) * g$$
-$$v_t := beta_2 * v_{t-1} + (1 - beta_2) * g * g$$
-$$variable := variable - lr_t * m_t / (\sqrt{v_t} + \epsilon)$$
+$$\text{lr}_t := \mathrm{lr} * \frac{\sqrt{1 - \beta_2^t}}{1 - \beta_1^t}$$
+$$m_t := \beta_1 * m_{t-1} + (1 - \beta_1) * g$$
+$$v_t := \beta_2 * v_{t-1} + (1 - \beta_2) * g * g$$
+$$\text{var} :=  \begin{cases} \text{var} - \text{lr}_t * \frac{m_t * \beta_1 + g * (1 - \beta_1)}{\sqrt{v_t} + \epsilon}, &\text{if use_nesterov} = True. \\\\ \text{var} \text{lr}_t * \frac{m_t}{\sqrt{v_t} + \epsilon}, &\text{otherwise} \\\\ \end{cases}$$
```

Comment:
The case is a win for me, but a \frac inside a case doesn't look as nice (feels like it hides the terms) , lets try some variants :)

How does 

```
$$
\begin{cases}
  \text{var} - (m_t \beta_1 + g (1 - \beta_1))\cdot\text{lr}_t/(\sqrt{v_t} + \epsilon), &\text{if use_nesterov}\\\\
  \text{var} - m_t \cdot \text{lr}_t /(\sqrt{v_t} + \epsilon), &\text{otherwise}
\end{cases}
$$
```

looks?

Reply:
It looks also ok.